index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Capstone Project 1</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+</head>
+<body>
+	<header class="container">
+		<div class="sub-container1">
+			<ul class="link1">
+				<li class="list-item"><a href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="list-item"><a href="#"><i class="fa fa-twitter"></i></a></li>
+				<li class="list-item"><a href="#">English</a></li>
+				<li class="list-item"><a href="#">My Page</a></li>
+				<li class="list-item"><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="sub-container2">
+			<div class="logo">
+				<a href="index.html"><img src="./images/logo2.png" alt=""></a>
+			</div>
+			<ul class="link2" id="nav">
+				<li class="list-item2 mobile"><a href="index.html">Home</a></li>
+				<li class="list-item2"><a href="about.html">About</a></li>
+				<li class="list-item2"><a href="#">Program</a></li>
+				<li class="list-item2"><a href="#">Join</a></li>
+				<li class="list-item2"><a href="#">Sponsor</a></li>
+				<li class="list-item2"><a href="#">News</a></li>
+			</ul>
+		</div>
+	</header>
+
+	<section class="homepage">
+		<div class="hamburger">
+			<div class="bars"></div>
+		</div>
+		<div class="container2">
+			<h1 class="header1">"Hello!"</h1>
+			<h2 class="header2">Health Awareness on the connection between Diabetes and Obesity Programme 2023</h2>
+			<p class="paragraph1">This program is aimed at bringing awareness to the masses, and educate them on how obesity can result in Diabetes Mellitus. A detailed explanation of Diabetes Mellitus and its side effects will be given. This way, thousands of lives are saved by this program, and the rate of occurrence of diabetes mellitus is reduced.</p>
+			<p class="paragraph2">2023.05.25(THU) ~ 2023.05.26(FRI)</p>
+			<p class="paragraph3">@ Abuja International Conference Centre, Abuja, Nigeria</p>
+		</div>
+	</section>
+
+	<section class="mainprogram">
+		<div class="container3">
+			<h2 class="header3">Main Program</h2>
```

Comment:
- [ ] You did a great job 😃 here at adding the line below the heading like the Behance, but I see that the color of this hr is different from the one in the Behance, so please 😊 try to put the correct color be like the Behance

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/226686593-6b2c06d4-ded1-402c-99c3-2c14f8cf1a8a.png)  |    ![image](https://user-images.githubusercontent.com/83360636/226686762-c8ed51cb-2143-4b99-b4f1-627311c1a93a.png) |